scripts/ci/run-packaging-tests.sh

```diff
@@ -14,7 +14,7 @@ VERSION=$(make -s version | cut -d '-' -f 1)
 TAG=$(make -s version | cut -d '-' -f 2- | tr '-' '.')
 for srpm in $(ls rpmbuild/SRPMS/skydive-${VERSION}*${TAG}.src.rpm)
 do
-    for root in centos-7-x86_64 epel-7-x86_64 fedora-27-x86_64 fedora-28-x86_64
```

Comment:
If I remember well centos mock was used to test some selinux package dependency, I think epel have container-selinux pkg but centos/rhel don't or something like that